### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 35

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#211] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152255008-8a604f61-ba98-446d-bfaf-5c520524a909.png)

**Comment by lord-marcel** [comment]
![es35_page-0001](https://user-images.githubusercontent.com/36630876/152649376-3c4576f6-86a4-41a2-ba8e-e66e0ae48d4d.jpg)
